src/Core/Domain/Customer/QueryResult/EditableCustomer.php

```diff
@@ -165,7 +171,8 @@ public function __construct(
         $website,
         $allowedOutstandingAmount,
         $maxPaymentDays,
-        $riskId
+        $riskId,
+        bool $isGuest
```

Comment:
To prevent bc breaks, you can maybe set this parameter as false by default?

Reply:
@PierreRambaud Whatever you think is best. Should I do it? @Progi1984

Reply:
That's also the value you pass by default in form

Reply:
@PierreRambaud Updated